### ISSUE TITLE
Torch Bug

### ISSUE THREAD

**Issue by ThePenguin14** [creepcomix/RealismExtreme_BedrockEdition#2]
Torches are bugged.
![Torch Image](https://user-images.githubusercontent.com/76607619/103328177-54ac0a00-4a1d-11eb-99d4-6f1a42669978.png)

**Comment by creepcomix** [comment]
Yeah, so, since the torches cannot have more than a 16x16 texture, I had to down-resolution the textures for fire and wood for the torches. Sorry for the inconvenience.
